build_scripts/rpm/mssql-cli.spec

```diff
@@ -81,6 +86,9 @@ mkdir -p %{buildroot}%{_bindir}
 printf "if [ -z ${PYTHONIOENCODING+x} ]; then export PYTHONIOENCODING=utf8; fi" > %{buildroot}%{_bindir}/mssql-cli
 printf '#!/usr/bin/env bash\n%{cli_lib_dir}/bin/python3 -Esm mssqlcli.main "$@"' > %{buildroot}%{_bindir}/mssql-cli
 
+# Some files got ambiguous python shebangs, we fix them after everything else is done
+pathfix.py -pni %{python_dir}/bin/python3 %{buildroot}/usr
```

Comment:
I this pathfix a fix we need to checkin, or a dependency we need to declare in the dev-requirements.txt

Reply:
Nope—it seems to ship either with RedHat or with Python. We declare a dependency with `BuildRequires` in the .spec file.

Reply:
Are you sure it's not [https://pypi.org/project/pathfix.py](https://pypi.org/project/pathfix.py)? Will BuildRequires install the package if not installed?

Reply:
That appears to be something else... despite the same name.